doc/Installation.md

```diff
@@ -0,0 +1,152 @@
+# INSTALL #
+
+This document explains how easy is to get tfchain up and running.
+
+## Using dockerfile ##
+
+## Using docker ##
+
+In this instalation we assume that you are working on a linux machine with docker installed.
```

Comment:
I don't think you have to assume that one is working on linux in order to install+run tfchain using the provided docker containers. As long as one has a host environment which is preconfigured with a working docker setup, this guide will work.

Because of that (but also we shouldn't try to explain basic things such as opening a terminal), we should just mention the relevant docker commands, and it will work.

Reply:
As you correctly noticed, the instructions are dumbed down, for most users it offers redundant information, but for the ones that lack the needed expertise, the instructions guide them.

And about the Linux, you are right, when I wrote it I was thinking on a set of instructions homogeneous that some knowledge can be applied on other parts.

Will make it more high level, more for experienced users. this is why I wanted a review.